code/modules/mob/mob_helpers.dm

```diff
@@ -295,18 +295,18 @@
 			to_chat(ghost, span_ghostalert(message))
 			continue
 
-		var/custom_link = enter_link ? " [enter_link]" : ""
-		var/link = " <a href='?src=[REF(ghost)];[action]=[REF(source)]'>([capitalize(action)])</a>"
+		var/interact_link = click_interact ? " <a href='?src=[REF(ghost)];play=[REF(source)]'>(Play)</a>" : ""
+		var/view_link = " <a href='?src=[REF(ghost)];view=[REF(source)]'>(View)</a>"
 
-		to_chat(ghost, span_ghostalert("[message][custom_link][link]"))
+		to_chat(ghost, span_ghostalert("[message][custom_link][interact_link][view_link]"))
 
 		var/atom/movable/screen/alert/notify_action/toast = ghost.throw_alert(
 			category = "[REF(source)]_notify_action",
 			type = /atom/movable/screen/alert/notify_action,
 		)
-		toast.action = action
 		toast.add_overlay(alert_overlay)
-		toast.desc = "[message] -- Click to [action]."
+		toast.click_interact = click_interact
+		toast.desc = "Click to [click_interact ? "play" : "view"]."
```

Comment:
Why are we getting rid of the mouseover desc message again? I just put that back :(

Reply:
Just personal preference I think. Some seemed redundant given the chat message. They also added the -- Click to (whatever), adding much more length. I tried replaced them with headers to compensate.

I'll do some side by sides for comparison

Reply:
for instance, it felt off that it had punctuation here then double dash. this same message works fine in chat btw but you see where the text artifact shows up
![image](https://github.com/tgstation/tgstation/assets/42397676/e0093758-99f1-45e5-a9bf-573e77fdeb5a)

Reply:
Fair enough. I don't REALLY think it's a big deal, especially now that there's a chance to read the chat output instead of just using the toast popup.